Worker cannot connect to RabbitMQ

Sentry worker cannot connect to rabbitmq, this cause that issues are not processed I guess .
```
11:36:23 [ERROR] celery.worker.consumer: consumer: Cannot connect to amqp://guest:**@sentry-rabbitmq:5672//: Error opening socket: hostname lookup failed.
```

--- a/sentry/templates/_helper.tpl
+++ b/sentry/templates/_helper.tpl
@@ -234,7 +234,7 @@ Set RabbitMQ host
 */}}
 {{- define "sentry.rabbitmq.host" -}}
 {{- if .Values.rabbitmq.enabled -}}
-    {{- default "rabbitmq"  (include "sentry.rabbitmq.fullname" .) -}}
+    {{- default "sentry-rabbitmq-ha"  (include "sentry.rabbitmq.fullname" .) -}}
 {{- else -}}
     {{ .Values.rabbitmq.host }}
 {{- end -}}

